objects/rct2ww/scenery_large/rct2.ww.pagodatw.json

```diff
@@ -92,7 +92,6 @@
             "pl-PL": "Dach pagody",
             "ru-RU": "Пагода",
             "pt-BR": "Torre Pagode",
-            "ja-JP": "パゴダ"
```

Comment:
No need to remove this.

Reply:
This PR should not touch Japanese at all, but some changes slipped through my initial filtered patch. Fixed now.